zebra-rpc/src/methods.rs

```diff
@@ -606,11 +595,7 @@ where
             let valid_addresses = address_strings.valid_addresses()?;
 
             let request = zebra_state::ReadRequest::AddressBalance(valid_addresses);
-            let response = state.oneshot(request).await.map_err(|error| Error {
-                code: ErrorCode::ServerError(0),
-                message: error.to_string(),
-                data: None,
-            })?;
+            let response = state.oneshot(request).await.map_err(to_error)?;
```

Comment:
Could we replace `.map_err(to_error)` in this file with `map_server_error()`, like:

```suggestion
            let response = state.oneshot(request).await.map_server_error()?;
```

Reply:
Done in [2c3a8db](https://github.com/ZcashFoundation/zebra/pull/8533/commits/2c3a8dbe8f3dd4a5bb256829c2e44f993d8d32de).